src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -116,5 +117,9 @@ namespace Microsoft.Terminal.TerminalControl
 
         Windows.Foundation.IReference<Windows.UI.Color> TabColor { get; };
         event Windows.Foundation.TypedEventHandler<Object, Object> TabColorChanged;
+
+        Boolean ReadOnly { get; };
+        void ToggleReadOnly();
+        event ReadOnlyChangedEventArgs ReadOnlyChanged;

```

Comment:
nit: should this be a typed handler like `event Windows.Foundation.TypedEventHandler<Object, Object> ReadOnlyChanged`? I feel like most of our recent events have been like that